source/PlanetPanel.cpp

```diff
@@ -125,12 +135,16 @@ void PlanetPanel::Draw()
 			info.SetCondition("has shipyard");
 
 		if(planet.HasOutfitter())
+		{
 			for(const auto &it : player.Ships())
 				if(it->GetSystem() == &system && !it->IsDisabled())
 				{
 					info.SetCondition("has outfitter");
 					break;
 				}
+			if(player.RelocationStatus() == PlayerInfo::RelocateStatus::TELEPORTED)
+				info.SetCondition("has outfitter");
```

Comment:
Why is this necessary?

Reply:
And even more important; if this is needed, then don't we also need it for things like trade for if the player has commodities in the Cargo that he/she needs to sell after relocation (when the player possibly has no flagship)?

Reply:
this is something temporary, i should fix that

Reply:
removed with next commit